com.unity.cinemachine/Runtime/Core/ConfinerOven.cs

```diff
@@ -281,12 +252,26 @@ public Vector2 Unstretch(Vector2 p)
         /// </summary>
         public BakedSolution GetBakedSolution(float frustumHeight)
         {
+            // If the user has set a max frustum height, respect it
+            frustumHeight = m_Cache.userSetMaxFrustumHeight <= 0
+                ? frustumHeight
+                : Mathf.Min(m_Cache.userSetMaxFrustumHeight, frustumHeight);
+            
+            // Special case: we are shrank to the mid point of the original input confiner area.
+            if (State == BakingState.BAKED && frustumHeight > m_Cache.theoriticalMaxFrustumHeight)
+            {
+                return new BakedSolution(
+                    m_AspectStretcher.Aspect, frustumHeight, false,
+                    m_PolygonRect, m_OriginalPolygon, 
+                    new List<List<IntPoint>>{new List<IntPoint> { m_MidPoint }});
+            }
```

Comment:
This is the new thing in the PR. 

What it does is that if the camera window is bigger (in both width and height) then the confining area. Then confine the camera to the mid point of the confining area.